doc/developer-notes.md

```diff
@@ -910,7 +910,35 @@ For development, it might be more convenient to verify all scripted-diffs in a r
 test/lint/commit-script-check.sh origin/master..HEAD
 ```
 
-Commit [`bb81e173`](https://github.com/bitcoin/bitcoin/commit/bb81e173) is an example of a scripted-diff.
+### Suggestions and examples
```

Comment:
nit: TOC could be updated as @promag [suggests](https://github.com/bitcoin/bitcoin/pull/17411#pullrequestreview-314052877).

Reply:
yes, will update the TOC